### DISCUSSION TITLE
Virtual workspace tool suggestion

### DISCUSSION THREAD

**Discussion by nazywamsiepawel** [abishekmuthian/startuptoolchain#9] [General]
Hi!

I'd like to suggest a tool that could be a nice addition to StartupToolchain. Disclaimer - I built it. More details below :)

Name: flat.social
Category: Virtual Workspace
Webpage: [https://flat.social](https://flat.social)
Why add it: While most of the tools are focused on productivity, flat.social aims to bring more joy to remote work through fun persistent virtual spaces where remote teams can hang out. There is a Slack integration and a free tier that fits a lot of small startups (up to 8 people).

Would love to hear your thoughts!

Best,
Paweł

**Comment by abishekmuthian** [comment]
Hi, thanks for reaching out.

It can be helpful if you can tell me what need-gap your service fills which others in that list don't?
